.cookiecutter.json

```diff
@@ -2,28 +2,29 @@
     "cookiecutter": {
         "codeowner_github_usernames": "@itdependsnetworks @jeffkala @qduk",
         "full_name": "Network to Code, LLC",
-        "email": "info@networktocode.com",
+        "email": "opensource@networktocode.com",
         "github_org": "networktocode",
         "description": "Common helper functions useful in network automation.",
         "project_name": "netutils",
         "project_slug": "netutils",
         "repo_url": "https://github.com/networktocode/netutils",
         "base_url": "netutils",
         "project_python_name": "netutils",
-        "project_python_base_version": "3.9",
+        "project_python_base_version": "3.10",
```

Comment:
I won't hold up.. but i am still of the opinion, we should be using the latest by default, and CI should run the oldest. There are some idiosyncrasies, like I think ruff says to use the oldest version in the config.. but that is just the config.

Reply:
ok, but the Dockerfile is using is, e.g. 

```
ARG PYTHON_VER="3.10"
```

also, where is the max defined?